bot/tests/test_remote.py

```diff
@@ -64,8 +64,8 @@ def test_baseline(mock_config, mock_revision, mock_workflow, mock_backend, mock_
     - an analyzer in failed status
     - with some issues in its log
     """
-    from code_review_bot.tasks.lint import MozLintIssue
     from code_review_bot.tasks.coverage import CoverageIssue
+    from code_review_bot.tasks.lint import MozLintIssue
```

Comment:
This change (done by pre-commit's isort) should already be on master, if you rebase (`git pull upstream master --rebase`) it will be picked up.